core/tests/widgets/test_imageview.py

```diff
@@ -46,8 +49,17 @@ def test_widget_created_with_args(app):
     assert widget.image == image
 
 
+def test_create_from_pil():
+    """An ImageView can be created from a PIL image"""
+    pil_img = PIL.Image.open(ABSOLUTE_FILE_PATH)
```

Comment:
We should use a context manager here so we don't leak file handles.

Reply:
Good catch! PIL should close files for single-frame images as soon as the data is read, but a) that isn't true for multiframe images like GIFs and b) it's probably better to be explicit in any event.